src/main/java/com/paneedah/weaponlib/EntityProjectile.java

```diff
@@ -243,7 +246,9 @@ public void onUpdate() {
              if(rtr != null) {
                  IBlockState state = compatibility.world(this).getBlockState(rtr.getBlockPos());
                  if(state.getMaterial() == Material.GLASS) {
-                     this.world.destroyBlock(rtr.getBlockPos(), true);
+                     if(!ModernConfigManager.penetratesGlass)
```

Comment:
Please replace this by something like `(!ModernConfigManager.penetratesGlass || ModernConfigManager.bulletBreakGlass)` as player might get confused why the glass isn't breaking when the config is set to true.